docs/content/en/docs/yaml-crd-ref/taskdefinition.md

```diff
@@ -67,10 +72,11 @@ spec:
   timeout: <duration-in-seconds>
 ```
 
-The API ref points to
+TODO: The API ref points to
 [Kubernetes doc](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#duration-v1-meta)
```

Comment:
if it helps this is the object from the K8s API https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Duration

Reply:
I slogged through both of those and still can't figure out what the unit of measure is for setting timeouts in k8s.  Am I missing something?

Reply:
I removed the TODO.  We can always add info later.